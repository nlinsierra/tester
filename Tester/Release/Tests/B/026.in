QD 5C AS 4C 3C 3D TD JD 6H 7D 3H 6C 3S AH 9D 8H 7C 4H KC TC 9H QS KH 7S 6D 5H 4S 6S TH 2D 4D 8D 7H JC 2S 8C KS JH JS QC AD 2H 8S AC
