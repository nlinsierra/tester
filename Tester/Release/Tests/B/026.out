8
2C 9C 5D KD QH 5S 9S TS
